src/main/java/com/voicepin/example/client/VoicepinClient.java

```diff
@@ -29,8 +29,8 @@
     private static final String VOICEPRINT_PATH = "voiceprints";
     private static final String PASSWORD_GROUP_PATH = "passwordGroups";
 
-    private static final String EXTERNAL_ID_QUERY = "extId";
-    private static final String API_KEY_QUERY = "apiKey";
+    public static final String USER_IDENTIFIER = "speakerId";
+    private static final String API_KEY = "apiKey";
```

Comment:
Spróbuj zmienić nazwę API_KEY. Obecna sugeruję że pod tą zmienną kryje się stałe apiKey, stara nazwa też nie była świetna jakby co. Może wydziel wszystkie te stałe do statycznej klasy i tą klasę nazwij tak aby było wiadomo co się w niej kryje. To tylko jedna z opcji.